src/providers/deno.rs

```diff
@@ -0,0 +1,37 @@
+use super::Provider;
+use crate::{nixpacks::app::App, providers::Pkg};
+use anyhow::Result;
+use regex::Regex;
+
+pub struct DenoProvider {}
+
+impl Provider for DenoProvider {
+    fn name(&self) -> &str {
+        "deno"
+    }
+
+    fn detect(&self, app: &App) -> Result<bool> {
+        if !app.includes_file("src/index.ts") {
+            false;
```

Comment:
```suggestion
            return false;
```

Reply:
`return Ok(false)` right?

Reply:
FWIW, `src/index{.ts,.js}` isn't always the entry-point. The line actually does nothing and doesn't need to do anything because of the code below. This could just be completely removed, as per my review below.